src/models/ElementUpdateData.ts

```diff
@@ -0,0 +1,36 @@
+import { Elements } from '@kontent-ai/delivery-sdk';
+import { ElementType } from '@kontent-ai/delivery-sdk/dist/cjs/elements/element-type';
+
+type ElementUpdateDataTemplate<TElement, TType extends ElementType> = {
```

Comment:
I'm unsure with the name here. If I get this right we group together Element data of "Deliver" shape with its type into one object. Have you consider to somehow persist "Deliver" in the name of the type? F.E `ElementUpdateDeliverDataTemplate` or smthing similar?

Reply:
I tried to avoid using the "Deliver" word in those types for several reasons:
- If I am not mistaken, "Deliver" is our internal term, publicly we only call it Delivery API or Delivery SDK. So it makes sense to use "Deliver" in our internal code, but I think it is not a good idea to add it to the public API.
- Even right now, these types are not exactly equal to Delivery SDK types. We only used parts of the original types, and in the future we may want to change something else as well. So we shouldn't create a strong naming connection between Delivery SDK types and these types.
What do you think?